src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpRequest.java

```diff
@@ -61,14 +61,11 @@ public void visit( final LogicalPlanVisitor visitor ) {
 	public String toString(){
 		final int codeOfReq = req.toString().hashCode();
 		final int codeOfFm = fm.getInterface().toString().hashCode();
```

Comment:
As these two variables are not used anymore in the rest of this function, you can remove them.